src/types.ts

```diff
@@ -6,8 +6,8 @@ export interface Chain {
     unit: string
   }
   safeConfirmations: number
-  isValidAddress: (address: string) => boolean
-  formatAddress: (address: string) => string
+  isValidAddress: (address: string, chain?: string, network?: string) => boolean
```

Comment:
`isValidAddress` is on the chain object, so I don't think it needs `chain` passed in?